### PR TITLE
Issue-159 resolved

### LINKED ISSUE #159: Add an image overlay on hover animation component
### Feature request-
Add an image overlay on hover animation component.

LGM-SOC 

<hr/>

**Note:**

-   If you want to work on an issue, you should check if it has already been assigned to anyone. **If the issue is free** you can comment `/assign` to get the issue assigned to you.
-   If you are raising a new issue and want to work on it then also you should comment `/assign` under the issue to get it auto assigned.
- It is adviced to take up/ work on only 2 issues at the same time.
-   Please **refrain from** adding labels to your issue/pull-request on your own. It is the job of the Project Admin and the Mentors to review your issue/pull-request and add labels accordingly.

### PR DESCRIPTION
**Pull request by puneetkaur2929**
## What is the change?
Add image overlay effects on hover component

## Related issue?
closes: #159 

## How was it tested?
Remove this line and tell us how the code was tested.

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

-   [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
-   [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/Mirage-UI/pulls) for the same update/change?
-   [] Have you made corresponding changes to the documentation?
-   [x] Have you tested the code before submission?
-   [x] Did you lint and format your code before making the Pull Request?
-   [] Have you made the GIF for the component you added?

## GIF you created:
Add here the GIF you created.

### CODE REVIEW

**Review by Pranav016 — CHANGES_REQUESTED** [review]
- Add a README for you component (use [README template](https://github.com/ALPHAVIO/Mirage-UI/blob/master/README_TEMPLATE.md)) and update the component [index](https://github.com/ALPHAVIO/Mirage-UI/blob/master/HTML%2BCSS%2BJS/README.md) as well

- try writing better commit messages that are descriptive about the code added.

**Review by theBeginner86 — CHANGES_REQUESTED** [review]
- Actually, there are some minor errors in the syntax (in the file:
`HTML+CSS+JS/Image-Animations/ImageOverlayEffects/README.md`).
You need to put enter after each tag you use so that the compiler can understand that you are making use of markdown.

- Rest everything looks good. 👍
Also, the different forms of overlay effect you choose are very reusable.

> Do test your README.md file before pushing your changes.

File: HTML+CSS+JS/Image-Animations/ImageOverlayEffects/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  - It could be used to display interesting hover effects on image as overlay.
+
+# Demo GIF for component:
+
+<div align="center">
+![ImageOverlayEffectsvideo -1](https://user-images.githubusercontent.com/80326308/125904828-23e6408e-4c1e-4364-99c1-d0e323e19daf.gif)
+![ImageOverlayEffectsvideo-2](https://user-images.githubusercontent.com/80326308/125906642-0dbad981-01ff-405c-89a4-496cdaf90ce5.gif)
+![ImageOverlayEffectsvideo-3](https://user-images.githubusercontent.com/80326308/125906118-367756c2-035b-4580-b5b6-5eb76ea5717b.gif)
+</div>
+ ## Setup instructions
```
Comment (theBeginner86) [inline HTML+CSS+JS/Image-Animations/ImageOverlayEffects/README.md]:
```suggestion
<div align="center">

![ImageOverlayEffectsvideo -1](https://user-images.githubusercontent.com/80326308/125904828-23e6408e-4c1e-4364-99c1-d0e323e19daf.gif)

![ImageOverlayEffectsvideo-2](https://user-images.githubusercontent.com/80326308/125906642-0dbad981-01ff-405c-89a4-496cdaf90ce5.gif)

![ImageOverlayEffectsvideo-3](https://user-images.githubusercontent.com/80326308/125906118-367756c2-035b-4580-b5b6-5eb76ea5717b.gif)

</div>

 ## Setup instructions
```

File: HTML+CSS+JS/Image-Animations/ImageOverlayEffects/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Ready to use.
+# Author
```
Comment (theBeginner86) [inline HTML+CSS+JS/Image-Animations/ImageOverlayEffects/README.md]:
```suggestion
Ready to use.

# Author
```

File: HTML+CSS+JS/Image-Animations/ImageOverlayEffects/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+![Puneet Kaur 😊](https://github.com/puneetkaur2929)
```
Comment (theBeginner86) [inline HTML+CSS+JS/Image-Animations/ImageOverlayEffects/README.md]:
```suggestion
[Puneet Kaur](https://github.com/puneetkaur2929) 
```
#### or

```suggestion
[Puneet Kaur](https://github.com/puneetkaur2929) 😊
```

**Review by theBeginner86 — CHANGES_REQUESTED** [review]

**Review by theBeginner86 — APPROVED** [review]
LGTM 👍

> From next time please consider writing better and descriptive commit messages. 🙂

**Comment by puneetkaur2929** [comment]
> LGTM 👍
> 
> > From next time please consider writing better and descriptive commit messages. 🙂

Glad that you liked it.
Ok I ll be careful from next time.

**Pull request merged**